docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,47 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work and therefore
+make it easier for everybody to find and access that specific
+information.
+
+.. note::
+
```

Comment:
I'd really like to reduce the amount of boxes we use in our docs. They're becoming too distracting.
This can be just a paragraph on its own, for example:
```
Feel free to suggest a new links to be added here, by creating an issue in our issue tracker.
```